go.mod

```
@@ -109,7 +109,7 @@ require (
 	go.opentelemetry.io/otel/trace v1.19.0 // @grafana/backend-platform
 	golang.org/x/crypto v0.14.0 // @grafana/backend-platform
 	golang.org/x/exp v0.0.0-20230321023759-10a507213a29 // @grafana/alerting-squad-backend
-	golang.org/x/net v0.17.0 // @grafana/grafana-bi-squad
+	golang.org/x/net v0.17.0 // @grafana/oss-big-tent
```

```suggestion
	golang.org/x/net v0.17.0 // @grafana/oss-big-tent @grafana/partner-datasources 
```

i recommend adding both squads, as we both use this dependency. WDYT?

---

These comments are parsed by `/scripts/modowners/modowners.go` which only supports one owner.

With both teams added it only returns the first one.
```
$ go run scripts/modowners/modowners.go owners -d golang.org/x/net@v0.17.0 go.mod
@grafana/oss-big-tent
```

I don't mind adding @grafana/partner-datasources anyway, if someone was to look it up manually.

---

i see.. hmm.. please add it in, at least if someone looks it up manually they'll know more.